From d8298fc54eb69d3989878be46e9c7d6ec2055650 Mon Sep 17 00:00:00 2001
From: Lee Duncan <lduncan@suse.com>
Date: Mon, 24 Sep 2018 16:22:18 -0700
Subject: [PATCH 05/87] Allow reading sysfs "port" to fail gracefully.

Do not consider it a fatal error if reading sysfs value
for "port" fails, since we allow failure for "address".
---
 libopeniscsiusr/session.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libopeniscsiusr/session.c b/libopeniscsiusr/session.c
index a833939..f8c89c3 100644
--- a/libopeniscsiusr/session.c
+++ b/libopeniscsiusr/session.c
@@ -205,8 +205,8 @@ int iscsi_session_get(struct iscsi_context *ctx, uint32_t sid,
 	_sysfs_prop_get_str(ctx, sysfs_con_dir_path, "address", (*se)->address,
 			    sizeof((*se)->address) / sizeof(char), "");
 
-	_good(_sysfs_prop_get_i32(ctx, sysfs_con_dir_path, "port",
-				  &((*se)->port), -1, false), rc, out);
+	_sysfs_prop_get_i32(ctx, sysfs_con_dir_path, "port",
+			    &((*se)->port), -1, true);
 
 	if ((strcmp((*se)->address, "") == 0) &&
 	    (strcmp((*se)->persistent_address, "") != 0))
-- 
1.8.3.1

